plugins/GSdx/GSTextureCache.cpp

```diff
@@ -75,9 +75,13 @@ void GSTextureCache::RemovePartial()
 
 	for (int type = 0; type < 2; type++)
 	{
-		for (auto &t : m_dst[type]) delete t;
-
-		m_dst[type].clear();
+		FastTargetPtrList& list = m_dst[type];
```

Comment:
I thought of that, but being paranoid I didn't do so: would it introduce overhead? (I think not).